style.css

```diff
@@ -15,15 +15,18 @@ html {
 
 body {
    font-size: 2rem;
-   background-color: #EEEEE0;
    font-family: 'Open Sans', sans-serif;
+   background-image: url('http://Setupshop/wp-content/uploads/2016/10/print011.jpg');
```

Comment:
Don't use absolute urls here. This will be a pain in the ass when you move your site to a production environment.

Also dont use files that you upload in WP in your theme. Just create an image folder and use `url('images/print011.jpg')`